apidef/oas/server.go

```diff
@@ -113,12 +111,15 @@ type GatewayTags struct {
 }
 
 func (gt *GatewayTags) Fill(api apidef.APIDefinition) {
-	gt.Enabled = api.EnableTags
+	gt.Enabled = !api.TagsDisabled
 	gt.Tags = api.Tags
 }
 
 func (gt *GatewayTags) ExtractTo(api *apidef.APIDefinition) {
-	api.EnableTags = gt.Enabled
+	if gt == nil {
```

Comment:
Why not consistency with the existing code ? The check outside was because of that it shows the coverage of function call or not. If such a change will be made, it should be done for all. It is not the topic of this PR. It opens an unnecessary discussion and slows the PR merge.

Reply:
I feel the change is valid. Given a nil *GatewayTags input, it sets the default values for APIDef tags; we enable tags by default to keep existing behaviour of the GW. Particularly the allocation is done to satisfy OAS validation.

I asume by consistency you mean that we should allocate the defaults in Server.ExtractTo? I'm not sure what your suggestion is here.

Reply:
nil check should be outside

Reply:
Done